### PR TITLE
Updating the "Implementing an inline widget" tutorial to new installation methods

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Updating the "Implementing an inline widget" tutorial to new installation methods.

---

### Additional information

Links to supporting repo won't work because PR is not merged to the master yet.
Here's PR to to the repo with the supporting files: https://github.com/ckeditor/ckeditor5-tutorials-examples/pull/10

### CODE REVIEW

File: docs/tutorials/widgets/implementing-an-inline-widget.md [hunk 1]
```diff
@@ -14,163 +14,30 @@ You will build a "placeholder" feature that allows the users to insert predefine
 	If you want to see the final product of this tutorial before you plunge in, check out the [demo](#demo).
 </info-box>
 
-## Before you start ⚠️
+## Before you start
 
 This guide assumes that you are familiar with the widgets concept introduced in the {@link tutorials/widgets/implementing-a-block-widget Implementing a block widget} tutorial. The tutorial will also reference various concepts from the {@link framework/architecture/intro CKEditor&nbsp;5 architecture}.
```
Comment (godai78) [inline docs/tutorials/widgets/implementing-an-inline-widget.md:19]: I wonder whether linking Widget guide wouldn't be better maybe? Will it be sufficient?

Comment (godai78) [inline docs/tutorials/widgets/implementing-an-inline-widget.md:19]: This section and the next section practically mention the same, it kind of looks redundant.

**Review by godai78 — APPROVED** [review]
LGTM

**Comment by Witoso** [comment]
Let's not wait for me, I will check all of them at one when on nightly.

File: docs/tutorials/widgets/implementing-an-inline-widget.md (hunk 1 above)
Comment (gorzelinski) [reply docs/tutorials/widgets/implementing-an-inline-widget.md]: Yeah, you're right

**Pull request merged**